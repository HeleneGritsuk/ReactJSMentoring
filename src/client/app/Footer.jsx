import React from 'react';

const Footer = () => (
  <footer>
    <div className="container">
      <p>
        React mentoring program 2018
      </p>
    </div>

  </footer>
);
export default Footer;
