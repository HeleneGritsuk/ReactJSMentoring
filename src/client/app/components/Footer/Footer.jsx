import React from 'react';

const Footer = () => (
  <footer>
    <div className="container">
      <p>
        netflixroulette
      </p>
    </div>

  </footer>
);
export default Footer;
