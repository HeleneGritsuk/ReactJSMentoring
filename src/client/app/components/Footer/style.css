footer {
  background: #464547;
  color: #ff8178;
}
