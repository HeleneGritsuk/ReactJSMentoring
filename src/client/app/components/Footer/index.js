import React from 'react';
import './style.css';

const Footer = () => (
  <footer>
    <div className="container">
      <p>
        netflixroulette
      </p>
    </div>

  </footer>
);
export default Footer;
