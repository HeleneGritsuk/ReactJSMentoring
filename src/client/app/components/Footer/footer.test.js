import React from "react";
import Footer from "./index";
import renderer from "react-test-renderer";

test("Footer is rendered normally", () => {
  const component = renderer.create(<Footer />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
