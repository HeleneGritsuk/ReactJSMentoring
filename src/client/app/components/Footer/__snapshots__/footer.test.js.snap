// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer is rendered normally 1`] = `
<footer>
  <div
    className="container"
  >
    <p>
      netflixroulette
    </p>
  </div>
</footer>
`;
